torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -2403,8 +2403,7 @@ def val_to_arg_str_for_prim_type(self, val, type_) -> str:
             else:
                 return "true" if val else "false"
         elif isinstance(val, int):
-            # uint64_t is long on Linux, but long long on MacOS and Windows
-            return f"{val}LL" if sys.platform in ["darwin", "win32"] else f"{val}L"
```

Comment:
Don't we want to keep this?  Otherwise large constants will be truncated on windows.

Reply:
Done.